src/PIL/TiffImagePlugin.py

```diff
@@ -1125,7 +1124,6 @@ def __getitem__(self, tag: int) -> Any:
         return val
 
 
-# undone -- switch this pointer when IFD_LEGACY_API == False
 ImageFileDirectory = ImageFileDirectory_v1
```

Comment:
Are there still plans to switch this in the future?

Reply:
There were something like 3 stages to the tiff IFD modernization, and I got through stage 1, years back now. Something like it is going to be required if we're going to do robust tiff tags, but it's not on my list anymore.

Reply:
I've restored the commit that it is 'undone' to 'switch this pointer'